src/metabase/api/card.clj

```diff
@@ -362,11 +363,11 @@
   "Compute a 'magic' cache TTL time (in seconds) for QUERY by multipling its historic average execution times by the `query-caching-ttl-ratio`.
    If the TTL is less than a second, this returns `nil` (i.e., the cache should not be utilized.)"
   [query]
-  (when-let [average-duration (qputil/query-average-duration query)]
+  (when-let [average-duration (query/average-execution-time-ms (qputil/query-hash query))]
```

Comment:
It'd be nice if the link from a card to it's avg execution time didn't require a hash. can we put this hash in the card table as well?
There is an implicit link, but it would be nice if we could join our way to nirvana here.